### ISSUE TITLE
Label weapon materials

### ISSUE THREAD

**Issue by StevenH237** [StevenH237/Synchrony-Switcheroo#26]
When playing NecroDancer normally, a player who can't see colors can still differentiate between weapon materials thanks to the fact that the description says which one it is. This isn't the case for weapons granted by Switcheroo, as the description is not seen when Switcheroo gives the item.

There should be an option to show the weapon's material name in the Attack box.

When used with other mods, this could even be useful for fully color-sighted players too, as nothing would stop two different mods from having the exact same weapon material colors, or close enough thereto to be hard to tell apart at speed.
